### DISCUSSION TITLE
两个月没更新了

### DISCUSSION THREAD

**Discussion by smallmj** [moesnow/March7thAssistant#308] [General]
这个项目被放弃了？

**Comment by bookbug666** [comment]
我也是這樣猜，但也只能等，等不及就找別的方案吧

**Comment by yhzcake** [comment]
src要单独装模拟器,电脑没地方了...
愁啊,啥时候能更新
不过看起来还是在管理的,所以应该不是弃项
不过拆分做不出来就先把其他小的地方打个包放出来呗..

**Comment by yhzcake** [comment]
喜大普奔，终于更新啦
